lara-typescript/src/section-authoring/api/lara-api-provider.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  IPage, PageId, ISection,
+  APIPageGetF, APIPagesGetF, IAuthoringAPIProvider,
+  ICreatePageItem
+} from "./api-types";
+
+const APIBase = "/api/v1";
+
+export const getLaraPageAPI = (host: string = "", activityId: string): IAuthoringAPIProvider => {
+
+  const prefix = `${host}/${APIBase}`;
```

Comment:
Just a note - this is going to generate double slashed urls like `https://app.lara.docker//api/v1` since `APIBase` starts with a `/`